app/src/main/res/values-tr/strings.xml

```diff
@@ -0,0 +1,56 @@
+<resources>
+    <string name="app_name">BetterCounter</string>
+    <string name="name_cant_be_blank">Sayaca bir ad ver</string>
+    <string name="already_exists">Bu ada sahip bir sayaç zaten mevcut</string>
+    <string name="counter_name">Sayaç adı</string>
+    <string name="decrease">Azalt</string>
+    <string name="increase">Artır</string>
+    <string name="never">Asla</string>
+    <string name="add_counter">Sayaç ekle</string>
+    <string name="cancel">İptal</string>
+    <string name="save">Kaydet</string>
+    <string name="edit_counter">Sayacı düzenle</string>
+    <string name="delete">Sil</string>
+    <string name="delete_or_reset">Sil/Sıfırla</string>
+    <string name="interval_day">Günlük</string>
+    <string name="interval_week">Haftalık</string>
+    <string name="interval_month">Aylık</string>
+    <string name="interval_year">Yıllık</string>
+    <string name="interval_lifetime">Ömür boyu</string>
+    <string name="time_ago_dhm">%1$dd %2$dh %3$dm önce</string>
+    <string name="time_ago_hm">%1$dh %2$dm önce</string>
+    <string name="time_ago_m">%1$dm önce</string>
+    <string name="time_in_dhm">%1$dd %2$dh %3$dm içinde</string>
+    <string name="time_in_hm">%1$dh %2$dm içinde</string>
+    <string name="time_in_m">%1$dm içinde</string>
+    <string name="just_now">Şu anda</string>
+    <string name="stats_average_per_hour">%1$.2f kez/saat"</string>
```

Comment:
Are the double quotes `"` after kez/saat intentional?

Reply:
Same for other lines below.

Reply:
I don't know. en strings file looks like this

Reply:
Oops, I think that's a mistake :) Thanks for helping me catch it and thanks for the translation :D